config/routes.rb

```diff
@@ -1,4 +1,19 @@
 Rails.application.routes.draw do
   devise_for :users
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  api_version(module: 'api/v1', path: { value: 'api/v1' }, defaults: { format: :json }) do
+    resources :users do
+        collection do
```

Comment:
Are you using 4 spaces for indentation? We are using just 2 on the rest of the project. Rubocop is only checking the app & spec folders